<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pycket</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="packetsList">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="startCaptureBtn"/>
   <addaction name="stopCaptureBtn"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_pcap_file"/>
    <addaction name="actionSave_as_pcap_file"/>
    <addaction name="exitBtn"/>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="startCaptureBtn"/>
    <addaction name="stopCaptureBtn"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCapture"/>
  </widget>
  <action name="startCaptureBtn">
   <property name="icon">
    <iconset>
     <normaloff>img/start.png</normaloff>img/start.png</iconset>
   </property>
   <property name="text">
    <string>Start capture</string>
   </property>
   <property name="toolTip">
    <string>Start a new capture</string>
   </property>
  </action>
  <action name="stopCaptureBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/stop.png</normaloff>img/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop capture</string>
   </property>
   <property name="toolTip">
    <string>Stop current capture</string>
   </property>
  </action>
  <action name="actionOpen_pcap_file">
   <property name="text">
    <string>Open pcap file</string>
   </property>
  </action>
  <action name="actionSave_as_pcap_file">
   <property name="text">
    <string>Save as pcap file</string>
   </property>
  </action>
  <action name="exitBtn">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit app</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
